CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.1.1]
+
+`2021-04-29`
+
+### What's new
+
+- **`DropdownMenu` now accepts `popoverProps`**.
+
+### Fixes
+
+- **Fixed image sizing issues in `ErrorPage`**.
+- **Fixed tooltip text blurring issues** through base CSS package.
```

Comment:
Should I close #25 now that don't need it? It could still help with something idk

Reply:
Also @veekeys @bentleyvk can you test if the blurriness is actually gone? I think it looks good on my machine but it's hard to tell

Reply:
Seems like it is fixed for me.
And about #25 it is now unnecessary and may bring unknown bugs, I would say to close it for now.

Reply:
It is fixed, but controlled story does not work for me.
Also, before release, need to fix wizard story, which got broken for some reason.